### DISCUSSION TITLE
My account marked as Spam, been more than 2+ months, and got no response from GitHub.

### DISCUSSION THREAD

**Discussion by neupkishor** [community/community#131843] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

What the hell is happening at GitHub now?
Nothing, nothing, nothing, and nothing as response till now.

My mistake, I had activated 2nd step authentication, had a really secure password, and someone hijacked my account without needing my password nor my verification code that's based on TOTP.

And, Got no emails for unauthorized login, and so more. No email, no email absolutely. Got my account marked as spam, requested for more than 2 months, and still the same.

Shit thing, hell lot of requesting and getting nothing in return, not even a status change.

**Comment by neupkishor** [comment]
So, solved.
Got my friend to make a case regarding the issue and GitHub team listened up to me in approximately a week.

**Reply by ashish07-07** [reply]
hi brother my account is also flagged can u please help me how to make it unflag . i am not ablr to raise ticket as well because i did not get teh verifcation code too  after sending my mobile number . can you please help me .

**Reply by ashish07-07** [reply]
can you please help me do the same please .
